server/area.py

```diff
@@ -469,7 +469,10 @@ def load(self, area):
             if self.music_ref == "":
                 self.clear_music()
         if self.music_ref != "":
-            self.load_music(f"storage/musiclists/{self.music_ref}.yaml")
+            if os.path.isfile(f"storage/musiclists/read_only/{self.music_ref}.yaml"):
+                self.load_music(f"storage/musiclists/read_only/{self.music_ref}.yaml")
+            else:
+                self.load_music(f"storage/musiclists/write_read/{self.music_ref}.yaml")
```

Comment:
remove `write_read/`, if it's not in a `read_only/` folder then it's a read and write.

Otherwise this will just break all servers currently using KFO